### DISCUSSION TITLE
Why are these kind of experiments and useless social media features not opt-in by default and why is there not opt-out?

### DISCUSSION THREAD

**Discussion by zearp** [community/community#54026] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This new feed is bad. Github is not a social network. When I click to show the people I follow that's what I want to see on all my devices. But instead I'm being shown the new feed regardless of my choice.

Why not add a setting to the user profile page and let us disable this. Or better, make these kind of features opt-in only. So the people who want to see strangers and unknown and new things in their feed can enable it. 

Don't enable it by default for everybody. It really isn't a new concept and I can guarantee that most people prefer to opt-in than opt-out. But we don't even have am opt-out. It simply does not exist.

Thanks Microsoft for ruining another product.

**Comment by dangolbeeker** [comment]
This brings the community together and I think it's a great feature.

**Reply by zearp** [reply]
By forcing it down everybody's throat. That's a nice way to get people onboard.

**Comment by sekelsta** [comment]
Why does this even exist? When I go to the github homepage I want to see my repositories. Not pointless distractions, that's the opposite of why I'm here.

**Reply by dangolbeeker** [reply]
Do this ...
1) Click my Avatar
2) Click follow 
3) Click your Avatar 
4) Go to your repo 
5) Be happy its a community
